### PR TITLE
Fix/inventory modal location

### PR DESCRIPTION
**Pull request by miguelstorani**
Fixed location overflow on inventory modal, added gateways circles when no location is available and fixed asset 360 link on modal.
image

### CODE REVIEW

File: src/components/inventory/inventory.modal.component.js [hunk 1]
```diff
@@ -21,123 +22,189 @@ class InventoryDetailsModal extends Component {
...
+    this.setState({isLoading: true});
+
+    this.props.commonStore.getData("inventory", paramsId).then((result) => {
+      if (result.status === HttpStatus.OK) {
+        this.setState({
+          gatewaysLocation: result.data.gateway_locations
+        }, () => this.setState({isLoading: false}));
+      }
+    });
+  }
```
Comment (miguelstorani) [inline src/components/inventory/inventory.modal.component.js:55]: this is where changes were made

File: src/components/inventory/inventory.modal.component.js [hunk 2]
```diff
@@ -186,33 +253,39 @@ class InventoryDetailsModal extends Component {
...
-                  <strong>Geolocation:</strong>
-                  <Geolocation location={item.location} />
+                  <strong>Location:</strong>
+                  <div style={{ height: "200px", width: "100%" }}>
+                    <Geolocation
+                      location={item.location}
+                      gatewaysLocations={this.state.gatewaysLocation}
+                      radius={2000}
+                    />
+                  </div>
```
Comment (miguelstorani) [inline src/components/inventory/inventory.modal.component.js:288]: changes

File: src/components/inventory/inventory.modal.component.js [hunk 3]
```diff
@@ -21,123 +22,189 @@ class InventoryDetailsModal extends Component {
...
+  UNSAFE_componentWillReceiveProps(newProps){
+    this.setState({
+      item: newProps.selectedItem.item,
+      activeIndex: newProps.selectedItem.index,
+    }, this.getGatewaysLocations);
+  }
+
+  getGatewaysLocations() {
+    const paramsId = {
+      type: this.props.selectedItem.item.type.toLowerCase(),
```
Comment (cpfarher) [inline src/components/inventory/inventory.modal.component.js:43]: If it is possible that type comes with undefined or null, you will have a problem here.... may be you can use something like this: 
```suggestion
      type: _.lowerCase(this.props.selectedItem.item.type),
```

**Review by cpfarher — APPROVED** [review]
LGTM - check a minor suggestion...

**Review by cpfarher — APPROVED** [review]
LGTM

**Pull request merged**